### ISSUE TITLE
Regex error with delete-obsolete

### ISSUE THREAD

**Issue by hatton** [crowdin/crowdin-cli#489]
`yarn crowdin upload sources --delete-obsolete` gives

```
$ crowdin upload sources --delete-obsolete
[OK] Fetching project info 
[ERROR] Unmatched closing ')' near index 10
^docs\(.+\\)?[^/]+\.[^/]+$
          ^
```

My `crowdin.yml` has 
```
    {
      "source": "/docs/**/*.*",
      "translation": /i18n/%two_letters_code%/docusaurus-plugin-content-docs/current/**/%original_file_name%,
      "skip_untranslated_files": true,
      "ignore": ["/**/*.png", "/**/*.jpg", "/**/*.gif"],
    },
```

I am using Windows, bash shell. project_id is 515854. @crowdin/cli version 3.8.1

**Comment by andrii-bodnar** [comment]
Hi @hatton, could you please try updating to the latest CLI version? **3.9.3**

This issue probably was fixed in the [3.9.0](https://github.com/crowdin/crowdin-cli/releases/tag/3.9.0) version

**Comment by hatton** [comment]
Thanks, the error is gone with the upgrade.
